pkg/packagekit/wix/service.go

```diff
@@ -16,15 +16,15 @@ type YesNoType string
 
 const (
 	Yes YesNoType = "yes"
-	No            = "no"
+	No  YesNoType = "no"
 )
 
 type ErrorControlType string
 
 const (
 	ErrorControlIgnore   ErrorControlType = "ignore"
-	ErrorControlNormal                    = "normal"
-	ErrorControlCritical                  = "critical"
+	ErrorControlNormal   ErrorControlType = "normal"
+	ErrorControlCritical ErrorControlType = "critical"
 )
 
 type StartType string
```

Comment:
I can't make this same fix for the `StartType` constants below because it causes a compilation error (!!) and I didn't want to make too many changes in this PR

Reply:
Reasonable! As long as it's not `nolint` we can fix it in a followup